fury/io.py

```diff
@@ -281,3 +282,51 @@ def save_polydata(polydata, file_name, binary=False, color_array_name=None):
         writer.SetFileTypeToBinary()
     writer.Update()
     writer.Write()
+
+
+def load_sprite_sheet(sheet_path, nb_rows, nb_cols, as_vtktype=False):
+    """Process and load sprites from a sprite sheet
+
+    Parameters
+    ----------
+    sheet_path: str
+        Path to the sprite sheet
+    nb_rows: int
+        Number of rows in the sprite sheet
+    nb_cols: int
+        Number of columns in the sprite sheet
+    as_vtktype: bool, optional
+        If True, the output is a vtkImageData
+
+    Returns
+    -------
+    Dict containing the processed sprites.
+    """
+    sprite_dicts = {}
+    sprite_sheet = Image.fromarray(load_image(sheet_path))
+    width, height = sprite_sheet.size
+
+    sprite_size_x = width // nb_rows
+    sprite_size_y = height // nb_cols
+
+    for row in range(nb_rows):
+        for col in range(nb_cols):
+            nxt_row = row + 1
+            nxt_col = col + 1
+
+            box = (row*sprite_size_x, col*sprite_size_y,
+                   nxt_row*sprite_size_x, nxt_col*sprite_size_y)
+
+            sprite_arr = sprite_sheet.crop(box)
+            if as_vtktype:
+                with InTemporaryDirectory() as tdir:
+                    tmp_img_path = os.path.join(tdir, f'{row}{col}.png')
+                    save_image(np.asarray(sprite_arr), tmp_img_path,
+                               compression_quality=100)
+
+                    sprite_dicts[f'{row}{col}'] = load_image(tmp_img_path,
+                                                             as_vtktype=True)
```

Comment:
strange to use the disk for this... Can you add a Todo and an issue saying that we need to replace this. 

Tag it as beginner-friendly

Reply:
I am using the disk to actually then load the saved image as `vtk.vtkImageData` and the same functionality is required in the `render_tex` method. I guess a function called `numpy_to_vtk_image_data` needs to be added to remove the need to write to disk.